#ifndef NatNET_ROS_MSGS_H
#define NatNET_ROS_MSGS_H

// include all the messages in msg folder
#include <NatRosPkg/Nat_msg.h>

#include <NatRosPkg/body.h>
#include <NatRosPkg/bodies.h>

#include <NatRosPkg/marker.h>
#include <NatRosPkg/markers.h>

#include <NatRosPkg/skeleton.h>
#include <NatRosPkg/skeletons.h>

#include <NatRosPkg/channel.h>

#include <NatRosPkg/forcePlate.h>
#include <NatRosPkg/forcePlates.h>

#include <NatRosPkg/device.h>
#include <NatRosPkg/devices.h>

#endif // NatNET_ROS_MSGS_H